packages/shared/watchPausable/index.md

```diff
@@ -7,6 +7,12 @@ alias: pausableWatch
 
 Pausable watch
 
+::: tip
+
+[Pausable Watcher](https://vuejs.org/api/reactivity-core.html#watch) has been added to Vue [since 3.5](https://github.com/vuejs/core/pull/9651), use `const { stop, pause, resume } = watch(watchSource, callback)` instead on supported versions.
+
+:::
+
```

Comment:
```suggestion
::: tip
[Pausable Watcher](https://vuejs.org/api/reactivity-core.html#watch) has been added to Vue [since 3.5](https://github.com/vuejs/core/pull/9651), use `const { stop, pause, resume } = watch(watchSource, callback)` instead, on supported versions.
:::

```

Reply:
The extra lines follows `toValue`'s doc and preserves readability in editor preview.
https://github.com/vueuse/vueuse/blob/c6def8b09839fd32c055594b648db47306fcdefa/packages/shared/toValue/index.md?plain=1#L12-L16
<img width="434" alt="image" src="https://github.com/user-attachments/assets/480f11f8-43b9-4ba5-8b07-21c442bba982" />

Removing them would result in confusion between tag and content.
<img width="450" alt="image" src="https://github.com/user-attachments/assets/94b464f6-6a66-423e-a7fb-b5b70f6bdc25" />